docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/DefaultStorageManagerExample.tsx

```diff
@@ -7,7 +7,6 @@ export const DefaultStorageManagerExample = () => {
       accessLevel="guest"
       maxFileCount={1}
       isResumable
-      provider="slow" // IGNORE
```

Comment:
Were these just not being used already?

Reply:
JS V6 removes the option to specify a provider.